src/pages/RepoPage/ComponentsTab/BackfillBanners/BackfillBanners.spec.jsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from 'custom-testing-library'
+
+import BackfillBanners from './BackfillBanners'
+import { useRepoBackfillingStatus } from './hooks'
+
+jest.mock('./TriggerSyncBanner', () => () => 'TriggerSyncBanner')
+jest.mock('./SyncingBanner', () => () => 'SyncingBanner')
+
+jest.mock('./hooks')
+
+describe('BackfillBanner', () => {
+  function setup(data) {
+    useRepoBackfillingStatus.mockReturnValue(data)
+
+    render(<BackfillBanners />)
+  }
+
+  describe('when rendered', () => {
+    describe('when components are not backfilled', () => {
+      it('displays TriggerSyncBanner component', () => {
+        setup({
+          componentsMeasurementsActive: false,
+          isRepoBackfilling: false,
+        })
+        expect(screen.getByText(/TriggerSyncBanner/)).toBeInTheDocument()
```

Comment:
@nicholas-codecov any reason for this in particular?

Reply:
rather, why is that necessary/good coding practice?

Reply:
More or less just consistency throughout the app ... also looks a bit cleaner imo, and since we're introducing new tests I'd prefer to "build for tomorrow" rather than today and try to keep things tidy than when I found them if able